### PR TITLE
[tests] Update CODEOWNERS

### PR DESCRIPTION
**Pull request by EndangeredMassa**
@mglagola: you mentioned not wanting to be listed as an owner here anymore.

@anatrajkovska: do you still want to be listed as an owner of this area?

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 *                                        @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood
 /.github/workflows                       @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood @ijjk
-/packages/cli/src/commands/domains       @mglagola @anatrajkovska
-/packages/cli/src/commands/certs         @mglagola @anatrajkovska
+/packages/cli/src/commands/domains       @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood
+/packages/cli/src/commands/certs         @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood
 /packages/cli/src/commands/env           @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood
```
Comment (TooTallNate) [inline .github/CODEOWNERS]: These are the same as `*` so I think we should just remove them:

```suggestion
```

**Review by mglagola — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (EndangeredMassa) [reply .github/CODEOWNERS]: done!

**Review by cb1kenobi — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -3,9 +3,6 @@
 
 *                                        @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood
 /.github/workflows                       @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood @ijjk
-/packages/cli/src/commands/domains       @mglagola @anatrajkovska
-/packages/cli/src/commands/certs         @mglagola @anatrajkovska
-/packages/cli/src/commands/env           @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood
```
Comment (styfle) [inline .github/CODEOWNERS:8]: I think `env` should have @baruchadi and the team working on env vars right now.

Comment (EndangeredMassa) [reply .github/CODEOWNERS:8]: I added them as a reviewer and reached out on Slack to confirm

**Review by anatrajkovska — APPROVED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (EndangeredMassa) [reply .github/CODEOWNERS:8]: I asked around and didn't get a response yet. I say we merge this and update it if they get back to me.

**Pull request merged**